src/annotator/guest.js

```diff
@@ -315,6 +327,37 @@ export default class Guest {
     });
   }
 
+  async _connectHostEvents() {
+    this._hostRPC.on(
```

Comment:
If the messages from the host could target a specific guest, that would remove the need for individual messages to filter based on frame identifier.

Reply:
Totally, agree.